cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -504,6 +507,9 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 				RuntimeEvent::EthereumOutboundQueue(snowbridge_pallet_outbound_queue::Event::MessageQueued {..}) => {},
 			]
 		);
+		let free_balance_of_sovereign_on_bh_after =
+			<BridgeHubRococo as BridgeHubRococoPallet>::Balances::free_balance(assethub_sovereign);
+		assert_eq!(free_balance_of_sovereign_on_bh_after, 955613334);
```

Comment:
As we may notice after the test there is some free balance left in the sov account.

Reply:
Addressed in https://github.com/Snowfork/polkadot-sdk/pull/167